src/models/user.ts

```diff
@@ -1,11 +1,6 @@
-interface User {
-  id: string,
-  name: string,
-  room: string,
-  readyToStart: boolean
-}
+import { IUser } from '../types'
 
-const users: User[] = []
+const users: IUser[] = []
```

Comment:
A good habit to get into is prefixing interfaces and types with `I`. https://github.com/bradzacher/eslint-plugin-typescript/blob/master/docs/rules/interface-name-prefix.md